l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -251,7 +251,7 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
     /// @notice the mailbox is called directly after the sharedBridge received the deposit
     /// this assumes that either ether is the base token or
-    /// the msg.sender has approved mintValue allowance for the sharedBridge.
+    /// the msg.sender has approved mintValue allowance for the nativeTokenVault.
```

Comment:
Here we should state that allowance is provided to NTV, but even if it has been provided to sharedBridge, it will be transferred. Same comment to be added for `requestL2TransactionTwoBridges`